internal/configuration/command.go

```diff
@@ -86,8 +86,8 @@ func getWhitelist(cfg *config.Config) map[string]*fieldInfo {
 	return whitelist
 }
 
-func isReadOnly(jsonField string) bool {
-	return jsonField == "disable_updates" && update.IsHomebrew()
+func isReadOnly(_ string) bool {
+	return false
 }
```

Comment:
Looks like this is a function and not a method of a struct... Any reason why we still need it / still need to pass a string to it?

Reply:
It's mostly pointless now, actually. It's only used to set the read only column of the output: `readOnly: isReadOnly(jsonTag),`.

I can remove this and just hardcode `false` until/unless we add an actual read-only config to the output.

Reply:
> I can remove this and just hardcode false

+1

Reply:
Oh I see, every configuration field is editable now. We could consider removing that column then?

Reply:
Yeah, sounds good. We can always reintroduce it if we ever feel the need to.